docs/Middleware.md

```diff
@@ -40,6 +40,8 @@ If you need to run a middleware only under certain path(s), just pass the path a
 
 *Note that this does not support routes with parameters, (eg: `/user/:id/comments`) and wildcards are not supported in multiple paths.*
 
+*Also note that wildcard path syntax is different from the fastify routes. (eg: `/css/(.*)` instead of `/css/*`)*
```

Comment:
I mean smt like that:

Also, note that the middleware syntax is different from the fastify route syntax. We use [middie](https://github.com/fastify/middie) for middlewares to support express.js.

Reply:
Ok, but I'll emphasize it as @RafaelGSS suggested. (you've reviewed the outdated version)

Reply:
It should be OK now.